pyaedt/misc/console_setup

```diff
@@ -8,43 +8,43 @@ cpython_console.py should be run instead of this script.
 This file can also serve as a template to modify PyAEDT scripts to take advantage of the command line arguments
 provided by the launcher
 """
-
 import sys
-
+import atexit
 from pyaedt import *
 import pyaedt
-import IPython
 pyaedt.settings.use_grpc_api=False
 
 aedt_process_id = sys.argv[1]
 version = sys.argv[2]
-with Desktop(
-			specified_version= version,
-			aedt_process_id = aedt_process_id,
-			new_desktop_session=False,
-			non_graphical=False,
-			close_on_exit=False,
-		) as desktop:
-    desktop.logger.info("Desktop class has been correctly initialized and connected to Desktop.")
-    desktop.logger.info(" ")
-
-    desktop.logger.info("===============================================================")
-    desktop.logger.info("ElectronicsDesktop {} Process ID {}".format(version, aedt_process_id))
-    desktop.logger.info("Cpython {}".format(sys.version))
-    desktop.logger.info("---------------------------------------------------------------")
-    desktop.logger.info(" - With Tab completion                                         ")
-    desktop.logger.info(" - dir()      - lists all available methods and objects       ")
-    desktop.logger.info(" - dir(obj)   - lists all available attributes/methods on obj  ")
-    desktop.logger.info(" - help(obj)  - provides available help on a method or object  ")
-    desktop.logger.info("Example: hfss = Hfss()  ")
-    desktop.logger.info("Example: m2d = Maxwell2d() ")
-    desktop.logger.info("Type exit() to close the console and release the desktop.  ")
-    desktop.logger.info("desktop object is initialized and available. Example: ")
-    desktop.logger.info("desktop.logger.info('Hello world')")
-    desktop.logger.info("===============================================================")
-    desktop.logger.info(" ")
-    desktop.logger.info(" ")
-    desktop.logger.info(" ")
-    IPython.embed(colors='Linux')
-    desktop.logger.info("Goodbye from PyAEDT Console")
+print("Loading PyAEDT Console")
+
+def release(d):
+    d.logger.info("Goodbye from PyAEDT Console")
+
+    d.release_desktop(False,False)
+
+
+desktop = Desktop(
+		specified_version= version,
+		aedt_process_id = aedt_process_id,
+		new_desktop_session=False,
+		non_graphical=False,
+		close_on_exit=False,
+	)
+print(" ")
+
+print("\033[92m****************************************************************")
+print("*  ElectronicsDesktop {} Process ID {}".format(version, aedt_process_id))
```

Comment:
Should there be a space in ElectronicsDesktop? It seems that to me that there should be.

Reply:
I'm reporting the same message is actually in AEDT Console.